Tests/Unit/DependencyInjection/HttplugExtensionTest.php

```diff
@@ -267,4 +268,124 @@ private function verifyProfilingDisabled()
             );
         }
     }
+
+    public function testClientShouldBePrivateByDefault()
```

Comment:
can we call this `testClientShouldHaveDefaultVisibility()` '

Reply:
Changed